api.c

```diff
@@ -572,10 +571,6 @@ int tcmu_emulate_evpd_inquiry(
 		/* Optimal xfer length */
 		memcpy(&data[12], &val32, 4);
 
-		/* MAXIMUM WRITE SAME LENGTH */
-		val64 = htobe64(VPD_MAX_WRITE_SAME_LENGTH);
-		memcpy(&data[36], &val64, 8);
-
```

Comment:
From some linux commit logs, I think it may due to that some devices could only support 2-Bytes block counts even with WRITE SAME(16). So set the MAXIMUM WRITE SAME LENGTH to 0xFFFF as default, and each device it could be configurable via sysfs.

If that's why the linux won't setting MAXIMUM WRITE SAME LENGTH to zero as defined in the spec, then the tcmu-runner emulator won't have that limitation, so could it be setting to zero as spec defined ?